README.md

```diff
@@ -1,2 +1,175 @@
-# Hello-Microverse
-Set up a "Hello Microverse" project Exersise
+<a name="readme-top"></a>
+
+<div align="center">
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Hello Miroverse] <a name="about-project"></a>
+
+**Hello Microverse** is a web application that shows a single title 'Hello Microverse' with a flavorable color. this web app is to show the basic structure of html and css.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://websitesetup.org/html-tutorial-beginners/">Html</a></li>
+    <li><a href="https://www.codecademy.com/learn/learn-css">Css</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="#">None</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="#">None</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+
+
+### Key Features <a name="key-features"></a>
+- **accessible heading**
+- **color full heading**
+- **smoth lokking**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://github.com/Paul-tes/Hello-Microverse)
```

Comment:
- [ ] I suggest you add the right link to a live demo of this project or add `coming soon` under the live demo section. Currently, the link takes a visitor to the repo. It will be more professional if the link takes visitors to the live demo or a note (`coming soon`) is left for visitors.

- [ ] It will be more professional if you add a summary of all the changes implemented on this repo to your PR description. Currently, I can only see one change. Kindly add a summary of all implemented changes, please.
![image](https://user-images.githubusercontent.com/94551484/226953340-b4a63b9f-7a60-4685-9a44-e21657fb5b49.png)